pipelines/basic.yaml

```diff
@@ -128,6 +128,10 @@ spec:
       type: string
       description: "Tag of the iqe image to run tests from"
       default: ""
+    - name: CUSTOM_PREPARATION_TASK_NAME
+      type: string
+      description: "Custom preparation task to run before running the tests"
+      default: "noop"
```

Comment:
Couldn't we use a `when` statement to skip the task entirely instead of running a `noop` task?